must-gather/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM quay.io/openshift/origin-cli:latest
+
+# copy all collection scripts to /usr/bin
+COPY collection-scripts/* /usr/bin/
```

Comment:
why specifically `collection-scripts` and not just `scripts`

Reply:
As in origin: https://github.com/openshift/must-gather/ I don't know if it is mandatory.

Reply:
I don't have a firm preference here but it seems verbose

Reply:
Will keep as is for now although more verbose to allow people recognize the structure and amend, e.g. third-party contributors.